contracts/truefi2/TrueFiPool2.sol

```diff
@@ -0,0 +1,392 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.6.10;
+
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {SafeMath} from "@openzeppelin/contracts/math/SafeMath.sol";
+
+import {ERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import {ERC20 as UpgradeableERC20} from "../common/UpgradeableERC20.sol";
```

Comment:
Keep only UpgradeableERC20 as ERC20. I don't think there's a need for both ERC20 imports